src/dlgProfilePreferences.cpp

```diff
@@ -631,6 +640,13 @@ void dlgProfilePreferences::setDisplayFont()
         pHost->mDisplayFont = font;
         if (mudlet::self()->mConsoleMap.contains(pHost)) {
             mudlet::self()->mConsoleMap[pHost]->changeColors();
+
```

Comment:
The preferences dialog is a mess already of instant application and application on save. I think we should get rid of the save button and just do instant application as it is the norm in modern applications now - so it's a mess we can leave be for later to fix.